src/components/button-toggle/package.json

```diff
@@ -1,3 +1,3 @@
 {
```

Comment:
Can we replace package.json with index.js?

Reply:
I thought the `package.json` was needed for the demo site? So it should get deleted when that gets trashed?

Reply:
Updated